tools/graphmap/test-data/result1_owler.mhap

```diff
@@ -0,0 +1,7 @@
+1 157 0.742839 3672 1 9 16450 22450 0 37 16202 16211
```

Comment:
Is this really tab separated? It looks space separated ...

Reply:
true. which format do u suggest to use?

Reply:
This looks rather like a matrix format of connectivity.. Should I use txt?

Reply:
yes, then go for text, or is this is proper fileform which we should support?

Reply:
I don't think it would become a popular format.  thank.